languages/language-python/src/main/kotlin/com/brackeys/ui/language/python/PythonLanguage.kt

```diff
@@ -29,9 +29,10 @@ class PythonLanguage : Language {
     companion object {
 
         private const val FILE_EXTENSION = ".py"
+        private const val FILE_EXTENSION_ALT = ".pyw"
```

Comment:
Also it would be great if you add this extension to `FileModel` class, so the file explorer could define this type of file as text

https://github.com/massivemadness/Brackeys-IDE/blob/master/filesystems/filesystem-base/src/main/kotlin/com/brackeys/ui/filesystem/base/model/FileModel.kt